app/build.gradle

```diff
@@ -119,7 +119,6 @@ android {
     testOptions {
         unitTests.returnDefaultValues = true
     }
-    buildToolsVersion project.ext.buildToolsVersion
```

Comment:
what was this for?

Reply:
it was twice, https://github.com/xmartlabs/bigbang/pull/156/files/566b8cd88d5f3356ad34467dc38642acd3fc1898#diff-39e7d8c00954e920b98e7636f0ac30b2L45